apiserver/facades/client/client/status.go

```diff
@@ -356,6 +360,40 @@ func (c *Client) FullStatus(args params.StatusParams) (params.FullStatus, error)
 			}
 			context.machines[status] = matched
 		}
+
```

Comment:
Consider breaking this out. `FullStatus` is too large.

Reply:
done.